### DISCUSSION TITLE
Welcome to Drain3 Discussions!

### DISCUSSION THREAD

**Discussion by davidohana** [logpai/Drain3#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghadesi** [comment]
Hi, why, in some cases, can we find some cluster ids that exist in the out of drain3, but if we check the result of the below code we cannot find those?
```python
...
print(f"--- Summary:")
drain_sorted_clusters = sorted(template_miner.drain.clusters, key=lambda it: it.size, reverse=True)

counter = 0
# The type of the cluster is <class 'drain3.cluster.Cluster'>
# (Referece: https://github.com/logpai/Drain3/blob/master/drain3/drain.py)
for cluster in drain_sorted_clusters:
    if cluster.size == 1:
        counter += 1
    logger.info(cluster)
logger.info(f"We have {counter} clusters with size 1.")
...
```

**Reply by ghadesi** [reply]
Because I want to create a ground truth file for my log file. Actually, I randomly pick some lines of log messages, and I want to compare them with the last version of cluster templates. 
![Screenshot 2023-02-15 at 9 20 57 PM](https://user-images.githubusercontent.com/6564410/219251327-127b8a6d-174b-41a3-9e2d-d310a5a519e6.png)

**Reply by Superskyyy** [reply]
Please create a separate discussion. this one is not for this purpose.
